### PR TITLE
[stable/kubernetes-dashboard]: Add networkPolicy

### PR DESCRIPTION
**Pull request by pho-enix**
Signed-off-by: Sebastian Poehn <sebastian.poehn@gmail.com>

#### What this PR does / why we need it:
Some Kubernetes environments feature a deny-all default policy. Then selective whitelistings of services are required. Optionally deploy such a networkPolicy to allowed service access.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/kubernetes-dashboard/templates/networkpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "kubernetes-dashboard.fullname" . }}
+  labels:
+    app: {{ template "kubernetes-dashboard.name" . }}
+    chart: {{ template "kubernetes-dashboard.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    kubernetes.io/cluster-service: "true"
```
Comment (kfox1111) [inline stable/kubernetes-dashboard/templates/networkpolicy.yaml]: Not sure it needs cluster-service here, or what effect that will have.

Comment (pho-enix) [reply stable/kubernetes-dashboard/templates/networkpolicy.yaml]: @kfox1111 Removed it. It only makes sense on Services.

**Comment by kfox1111** [comment]
Looks like it might need a version bump now, then looks good to me.

**Comment by pho-enix** [comment]
@kfox1111 Yep. Done.

**Review by kfox1111 — COMMENTED** [review]
/lgtm

**Pull request merged**